### PR TITLE
Renamed listing.js to JSX

### LINKED ISSUE #687: Use JSX for listing.js
This would let us get rid of the loader functions and use React components directly from within listing.jsx.

### PR DESCRIPTION
**Pull request by noisecapella**
This just allows us to use the JSX syntax within listing.js (renamed to listing.jsx)

Fixes #687

### CODE REVIEW

File: ui/templates/repository.html [hunk 1]
```diff
@@ -66,6 +65,9 @@
 <script type="text/jsx"
         src="{% static "ui/js/utils.jsx" %}">
 </script>
+<script type="text/jsx"
+        src="{% static "ui/js/listing.jsx" %}">
```
Comment (jamiefolsom) [inline ui/templates/repository.html:69]: Is this indentation standard? Looks like 8 spaces.

Comment (noisecapella) [inline ui/templates/repository.html:69]: It matches everything else in the file, but there's no particular standard for HTML indentation in LORE

Comment (jamiefolsom) [inline ui/templates/repository.html:69]: Oh, I see the attribute names are horizontally aligned.

Comment (jamiefolsom) [inline ui/templates/repository.html:69]: Looks fine to me

**Pull request merged**